### ISSUE TITLE
Android status bar icons black on black

### ISSUE THREAD

**Issue by mvsde** [mattermost/mattermost-mobile#893]
#### Summary
If the `sidebarHeaderBg` color is too light, the Android status bar icons will be rendered black on a black status bar.

#### Environment Information

- Device Name: OnePlus One
- OS Version: Android 7.1.2
- Mattermost App Version: 1.2.0 (Build 49)
- Mattermost Server Version: 4.1.0

#### Steps to reproduce
* Change to a custom theme.
* Set the Sidebar Header BG (`sidebarHeaderBg`) to something light, for example `#14c4a9`.

#### Expected behavior
The Android status bar icons should be visible.

#### Observed behavior
While loading everything is fine:
![loading](https://user-images.githubusercontent.com/3825202/30281783-a1605ce0-9713-11e7-9adc-2f3845ec07f3.png)

Once the theme has loaded, the icons disappear:
![loaded](https://user-images.githubusercontent.com/3825202/30281784-a1617760-9713-11e7-8e78-7893dcb72edd.png)

#### Possible fixes
The best solution would be a colored status bar to follow the Material Design guidelines.

A quick fix might be having always white icons.

**Comment by enahum** [comment]
I'm not able to reproduce this on an Android 7.1.1 nor a 6.0.1, can you share more details?

**Comment by mvsde** [comment]
Maybe it helps if I provide the full theme:

```json
{"awayIndicator":"#ffbc42","buttonBg":"#14c4a9","buttonColor":"#ffffff","centerChannelBg":"#ffffff","centerChannelColor":"#3c3c3c","codeTheme":"github","errorTextColor":"#fd5960","linkColor":"#14c4a9","mentionBj":"#c522d3","mentionColor":"#ffffff","mentionHighlightBg":"#c522d3","mentionHighlightLink":"#ffffff","newMessageSeparator":"#777777","onlineIndicator":"#06d6a0","sidebarBg":"#3c3c3c","sidebarHeaderBg":"#14c4a9","sidebarHeaderTextColor":"#ffffff","sidebarText":"#ffffff","sidebarTextActiveBorder":"#14c4a9","sidebarTextActiveColor":"#ffffff","sidebarTextHoverBg":"#0d7f6e","sidebarUnreadText":"#ffffff"}
```

I also asked a co-worker to try the theme and he was able to reproduce this problem.

**Comment by enahum** [comment]
I just tested it and this is what I get.
![screenshot_20170919-103321](https://user-images.githubusercontent.com/6757047/30594918-06456520-9d26-11e7-9905-b98d27ff0c33.png)

**Comment by finn-h** [comment]
I am able to reproduce this. Notice the colored battery indicator.

Sony Xperia Z5
Android 7.1.1
App version 1.2.0

![image](https://user-images.githubusercontent.com/6942107/30609649-4a334cd8-9d7c-11e7-9c3a-f196629b3186.png)

**Comment by enahum** [comment]
@finn-h @mvsde I've submitted a PR #922 that should fix this issue

**Comment by mvsde** [comment]
That's awesome. Thanks a lot! :-)

**Comment by enahum** [comment]
Closed via #922
